source/administration/command-line-tools.rst

```diff
@@ -2234,7 +2235,29 @@ mattermost webhook modify-outgoing
           --icon [iconURL]              Icon URL 
 	  --url [callbackURL]           Callback URL 
 	  --content-type string         Content type 
+	  
+mattermost webhook move-outgoing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+  Description
+    Move an existing outgoing webhook to another team and channel by by specifying its id. If the webhook to be moved has a channel id, it must have a new one specified when moving to a new team. 
```

Comment:
```suggestion
    Move an existing outgoing webhook to another team and channel by specifying its ID. If the webhook you're moving already has a channel ID, a new one must be specified when moving it to the new team. 
```

Reply:
Question: I may not understand the context but the example below doesn't look like it includes a new ID when moving it to a new channel. Also, is it possible for a webhook not to have an ID (i.e., "if it already has an ID") or are we saying that a webhook has an ID but can also have a channel ID?

Reply:
@justinegeffen I rewrote the description, does that clarify the requirements on the channel ID for you?  Also updated the example to be more explicit on the webhook ID and channel id/name.  Let me know if this is better?

Reply:
That's awesome, thank you. :)